development/secrets-rotator/cloud-run/service-account-keys-cleaner/README.md

```diff
@@ -0,0 +1,32 @@
+# Cleanup of service account secrets
+
+## Overview
+
+The Cloud Run service deletes old keys for a GCP service account and updates the required secret data for all service account secrets stored in the Secret Manager. The service is triggered by a Cloud Scheduler job.
+
+1. Cloud Scheduler calls service-account-keys-cleaner service.
+2. For each secret stored in Secret Manager service executes the following steps:
+    1. Checks if the value of the **type** label is set to `service-account`. If not, it stops running.
+    2. Checks if the value of the **skip-cleanup** label is not set to `true`. If not, it stops running.
```

Comment:
It might be easier to follow if you say `(...) is set to `true`. If it is, the service stops running.`

Reply:
corrected.